src/main/java/nextstep/ladder/model/Person.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.ladder.model;
+import java.util.List;
+
+public class Person {
+    private String name;
+    private int position;
+
+    public Person(String name, int position){
+        this.name = name;
+        this.position = position;
+    }
+    public String getName(){

```

Comment:
메서드 구분을 위해 메서드 사이에 한칸 띄어주시면 좋을 것 같아요 :)
```suggestion
    }
    
    public String getName() {

```